addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -136,6 +136,17 @@ var SnippetEditor = Widget.extend({
         // a flickering when not needed.
         this.$target.on('transitionend.snippet_editor, animationend.snippet_editor', postAnimationCover);
 
+        // Disable the pointer events on the handle during the mouse wheel event.
+        // To prevent page scrolling from stopping when hovering over a handle.
+        const $handles = this.$el.find('.o_handle');
+        $handles.on('wheel.snippet_editor', _.throttle(() => {
```

Comment:
As explained before I don't think you need to attach an event handler on all handles. Just on the main overlay element (`this.el` here) and you can then use the `events` property with a real event handler definition, etc.